### PR TITLE
Added conda capitalization standards to CONTRIBUTING.md.

### PR DESCRIPTION
**Pull request by kathatherine**
### Description

 
Added conda capitalization standards to CONTRIBUTING.md.

### Checklist - did you ...

- [x] Add a file to the `news` directory ([using the template](../blob/main/news/TEMPLATE)) for the next release's release notes?
     
- [x] ~Add / update necessary tests?~
- [x] Add / update outdated documentation?

### CODE REVIEW

**Review by jezdez — DISMISSED** [review]
Just some nits and a suggestion.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -81,3 +81,29 @@ To see how this process works for this project, read "[The Triaging Process at c
 [anaconda-bug-report]: https://anaconda.org/contact/report
 [triaging]: https://github.com/conda/infra/blob/main/TRIAGING.md
 [development-environment]: https://docs.conda.io/projects/conda/en/latest/dev-guide/development-environment.html
+
+## Conda capitalization standards
+
+Conda should be written in lowercase, whether in reference to the tool, ecosystem, packages, or organization. References to the conda command should use code formatting (i.e. `conda`). If the use of conda is not a command and if conda is at the beginning of a sentence, conda should be uppercase.
```
[the remark this answers is not in the document]
Comment (kathatherine) [reply CONTRIBUTING.md:87]: That's a great idea.

**Comment by kathatherine** [comment]
Not sure what I did to make that happen ^

I'm gonna write up a quick release note for this also.

**Review by kenodegard — APPROVED** [review]

**Comment by kathatherine** [comment]
pre-commit.ci autofix

**Review by kenodegard — APPROVED** [review]

**Pull request merged**